### PR TITLE
changes into cli-fang-cli-defang function

### PR DESCRIPTION
**Pull request by dhruvdabhi101**
Check this PR and Suggestions are welcome.
Changed cli_fang and cli_defang functions solving issue no #91

### CODE REVIEW

File: ioc_fanger/ioc_fanger.py [hunk 1]
```diff
@@ -31,12 +31,13 @@ def fang(text: str, debug=False):
 @click.argument("text", required=False)
 def cli_fang(text):
     """CLI interface for fanging indicators."""
-    stdin_text = click.get_text_stream("stdin")
-
+
```
Comment (fhightower) [inline ioc_fanger/ioc_fanger.py:34]: Let's remove this blank line as it's not necessary:

```suggestion
```

File: ioc_fanger/ioc_fanger.py [hunk 2]
```diff
@@ -56,12 +57,13 @@ def defang(text):
 @click.argument("text", required=False)
 def cli_defang(text):
     """CLI interface for defanging indicators."""
-    stdin_text = click.get_text_stream("stdin")
-
+
```
Comment (fhightower) [inline ioc_fanger/ioc_fanger.py:60]: Let's remove this blank line as it's not necessary:

```suggestion
```

File: ioc_fanger/ioc_fanger.py [hunk 3]
```diff
@@ -31,12 +31,13 @@ def fang(text: str, debug=False):
...
 def cli_fang(text):
     """CLI interface for fanging indicators."""
-    stdin_text = click.get_text_stream("stdin")
-
+    
     if text:
         fanged_text = fang(text)
         print(fanged_text)
-    elif stdin_text:
+        return
```
Comment (fhightower) [inline ioc_fanger/ioc_fanger.py:38]: I recommend adding a newline here so it is easier for readers to see the two different paths we can take in this function (one path if there is text and another if there is stdin_text):

```suggestion
        return

```

File: ioc_fanger/ioc_fanger.py [hunk 4]
```diff
@@ -56,12 +57,13 @@ def defang(text):
...
 def cli_defang(text):
     """CLI interface for defanging indicators."""
-    stdin_text = click.get_text_stream("stdin")
-
+    
     if text:
         defanged_text = defang(text)
         print(defanged_text)
-    elif stdin_text:
+        return
```
Comment (fhightower) [inline ioc_fanger/ioc_fanger.py:64]: I recommend adding a newline here so it is easier for readers to see the two different paths we can take in this function (one path if there is text and another if there is stdin_text):

```suggestion
        return

```

**Review by fhightower — COMMENTED** [review]
Wonderful, thanks @dhruvdabhi101! This PR is great! I've made a few suggestions which either remove and add an empty line to improve readability. Please review and add those suggestions (or ask questions if they don't make sense) and then I'll approve.

Thanks again for the great PR!

**Comment by dhruvdabhi101** [comment]
Please check and tell if there is still something remaining.

**Review by fhightower — APPROVED** [review]
Looks great - thanks @dhruvdabhi101 ! I'll merge this shortly 🎉

**Pull request merged**